docroot/profiles/custom/cgov_site/modules/custom/cgov_core/cgov_core.install

```diff
@@ -145,41 +145,48 @@ function _cgov_core_install_permissions(CgovCoreTools $siteHelper) {
  *
  * @see system_themes_installed()
  */
-function cgov_core_themes_installed() {
+function cgov_core_themes_installed($theme_list) {
```

Comment:
A separate issue was created, #4172. Just update this commit to:
1. come first in the commit list
2. update the commit message to reference the new number.